src/EditorFeatures/Core.Cocoa/Microsoft.CodeAnalysis.EditorFeatures.Cocoa.csproj

```diff
@@ -39,6 +40,10 @@
     <PublicAPI Include="PublicAPI.Shipped.txt" />
     <PublicAPI Include="PublicAPI.Unshipped.txt" />
   </ItemGroup>
+  <ItemGroup>
+    <RestrictedInternalsVisibleTo Include="Microsoft.VisualStudio.LanguageServices.TypeScript" Key="$(TypeScriptKey)" Partner="VSTypeScript" />
+    <RestrictedInternalsVisibleTo Include="Microsoft.CodeAnalysis.TypeScript.EditorFeatures" Key="$(TypeScriptKey)" Partner="VSTypeScript" />
```

Comment:
@uniqueiniquity hopefully these assembly names are still correct for VS for Mac?

Reply:
Nope - will need `TypeScriptAddin` added.

Reply:
The MS.VS.LS.TS one is VS-windows-only.

Reply:
Just `TypeScriptAddin`? or `Microsoft.VisualStudio.LanguageServices.TypeScriptAddin`?

Reply:
Yeah, just `TypeScriptAddin`. It's a really goofy name.

Reply:
Thanks, updated :)